mypy/messages.py

```diff
@@ -1113,7 +1113,7 @@ def overloaded_signature_will_never_match(self, index1: int, index2: int,
             context)
 
     def overloaded_signatures_typevar_specific(self, index: int, context: Context) -> None:
-        self.fail('Overloaded function implementation cannot satisfy signature {} '.format(index) +
+        self.fail(f'Overloaded function implementation cannot satisfy signature {index} ' +
```

Comment:
It might be nice to make this line PEP8-compliant by moving the `+` at the end of the line to the beginning of the next line

Reply:
We should merge the PR that Blackens mypy so we don't have to worry about this.

Reply:
That too :)